DatadogCore/Sources/Core/Storage/Storage+TLV.swift

```diff
@@ -0,0 +1,25 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+
+/// Default max data length in TLV block for batch file (safety check) - 10 MB
+internal let MAX_DATA_LENGTH: UInt64 = 10 * 1_024 * 1_024
```

Comment:
There's an extension in `DatadogInternal` that allows doing `10.MB.asUint64()`

Reply:
Oh, I was looking of sth this 👍. I'll do it the next PR for `DataStore+TLV` where I am revisiting this part.